### PR TITLE
Fix isPointInPath Path Parameter support info for Safari

### PR DESCRIPTION
**Pull request by ramhugger**
This Pull Request is the continuation of and closes #9353.
I changed Safari isPointInPath Path Parameter support info from false to "7", as suggested by @foolip in the original Pull Request #9353.

### CODE REVIEW

File: api/CanvasRenderingContext2D.json [hunk 1]
```diff
@@ -2320,7 +2320,7 @@
                 "version_added": "24"
               },
               "safari": {
-                "version_added": false
+                "version_added": "7"
               },
               "safari_ios": {
                 "version_added": null
```
Comment (ddbeck) [inline api/CanvasRenderingContext2D.json:2326]: Hi @ramhugger. Based on the discussion on the previous PR, I think you could apply this change here too, then this PR could fully replace the previous one:

```suggestion
                "version_added": "7"
```

**Review by ddbeck — CHANGES_REQUESTED** [review]

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Comment by ramhugger** [comment]
It's me to thank you @foolip and @ddbeck 🥳!

**Comment by foolip** [comment]
@ramhugger your change prompted me to take a closer look at some of the other entries, and I ended up changing a bunch more to be "7" in https://github.com/mdn/browser-compat-data/pull/9433.

**Pull request merged**